### DISCUSSION TITLE
Support for decentralized git forge Radicle

### DISCUSSION THREAD

**Discussion by worldofgeese** [GitJournal/GitJournal#675] [Feature Requests]
[Radicle](https://radicle.xyz/) requires the `rad` binary for most actions so I assume the binary would need to be included but it'd be great to see GitJournal support this forge!

**Comment by vHanda** [comment]
This is a wontfix from me.

It's way way out of my use case, and GitJournal is heavily integrated in concepts of Git (immutability, sha1 hash, etc). Plus, I can't call binaries in ios.
